aeron-system-tests/src/test/java/io/aeron/SpySimulatedConnectionTest.java

```diff
@@ -276,6 +278,36 @@ else if (!isSubscriptionClosed)
         assertEquals(messagesToSend, fragmentCountSpy.value);
     }
 
+    @Test
+    @InterruptAfter(10)
+    void shouldNotHaveShortSendsWithMDCPublication() throws Exception
+    {
+        launch();
+
+        final String channel = "aeron:udp?control=localhost:40456|control-mode=dynamic";
+
+        spy = client.addSubscription(spyForChannel(channel), STREAM_ID);
+        publication = client.addExclusivePublication(channel + "|ssc=true", STREAM_ID);
+
+        Tests.awaitConnected(spy);
+        Tests.awaitConnected(publication);
+
+        Thread.sleep(500);
```

Comment:
Why sleep? Why not exchange a single message instead? This would not be clock dependent.

Also, if you need to wait for something then wait for `Bytes sent` to be incremented in addition to observing the data by the spy.

Reply:
I need `aeron_network_publication_setup_message_check` to be called, so I need the sender thread to loop through at least once.  FWIW, if I remove the sleep, the test still seems to 'work' - i.e. without the fix to the C media driver, the subsequent assert fails every time, indicating we got at least one short send.  Seems like it's possible that by the time both the spy and the publication are 'connected', the sender thread must have already already run through a few times.  But I wasn't 100% sure about that, looking through the code.